github/Organization.py

```diff
@@ -571,33 +571,42 @@ def create_secret(
         secret_name: str,
         unencrypted_value: str,
         visibility: str = "all",
+        secret_type: str = "actions",
```

Comment:
Adding this here might break some user code, better add this as the last argument.